Makefile

```
@@ -203,10 +203,9 @@ distclean: clean
 	$(MAKE) -C tools distclean
 
 setup:
-	$(MAKE) -C tools
-	python3 fixbaserom.py
-	python3 extract_baserom.py
-	python3 extract_assets.py
+	$(MAKE) -C tools -j
+	mkdir -p baserom
+	tools/extract_baserom
```

What happened to `fixbaserom.py` and `extract_assets.py`?

```suggestion
	$(MAKE) -C tools
	mkdir -p baserom
	tools/extract_baserom
```